eclair-core/src/main/scala/fr/acinq/eclair/PluginParams.scala

```diff
@@ -55,4 +58,15 @@ trait CustomCommitmentsPlugin extends PluginParams {
    * returned by this method.
    */
   def getHtlcsRelayedOut(htlcsIn: Seq[IncomingHtlc], nodeParams: NodeParams, log: LoggingAdapter): Map[Origin, Set[(ByteVector32, Long)]]
+}
+
+// @formatter:off
+case class InterceptOpenChannelReceived(replyTo: ActorRef[InterceptOpenChannelResponse], open: Either[OpenChannel, OpenDualFundedChannel], temporaryChannelId: ByteVector32, localParams: LocalParams, fundingAmount: Satoshi)
```

Comment:
nit: `fundingAmount` is redundant since it's already in `open_channel` and `open_channel2`? That could instead be a property of the case class:

```suggestion
case class InterceptOpenChannelReceived(replyTo: ActorRef[InterceptOpenChannelResponse], open: Either[OpenChannel, OpenDualFundedChannel], temporaryChannelId: ByteVector32, localParams: LocalParams) {
  val remoteFundingAmount: Satoshi = open match {
    case open: OpenChannel => open.fundingSatoshis
    case open: OpenDualFundedChannel => open.fundingAmount
  }
}
```

Reply:
Great suggestion! Fixed in [37eadd5](https://github.com/ACINQ/eclair/pull/2552/commits/37eadd56a1032d0c890abe0608d457df17496e4d).